modin/pandas/test/test_io.py

```diff
@@ -370,8 +378,9 @@ def teardown_fwf_file():
         os.remove(TEST_FWF_FILENAME)
 
 
-def test_read_csv_gzip():
-    gzip_path = "modin/pandas/test/data/test_df.csv.gz"
+def test_read_csv_gzip(make_csv_file):
```

Comment:
Can we move these to be together with the other `test_from_csv` tests and rename to `test_from_csv_{}`?

https://github.com/modin-project/modin/blob/12a895195a51b7846fe16e46195b9ee6beede2ec/modin/pandas/test/test_io.py#L983

https://github.com/modin-project/modin/blob/12a895195a51b7846fe16e46195b9ee6beede2ec/modin/pandas/test/test_io.py#L669

https://github.com/modin-project/modin/blob/12a895195a51b7846fe16e46195b9ee6beede2ec/modin/pandas/test/test_io.py#L669